src/commands/print.yml

```diff
@@ -0,0 +1,12 @@
+description: >
+  Print a message to console.
+
+parameters:
+  message:
+    description: "Message to print to console"
+    type: string
+    default: "Empty message"
+
+steps:
+  - run: |
+      echo "<< parameters.message >>"
```

Comment:
Why?

Reply:
true - can be inlined 🤦‍♂️ 
i was copy pasting too much here